### PR TITLE
Fix/journal assignments

### PR DESCRIPTION
**Pull request by melisabok**
- Send email notifications on public comments.
- Re write email notifications based on @zbialecki feedback.
- Fix link construction.
- Check conflicts before any edge assignment.

### CODE REVIEW

File: openreview/journal/journal.py [hunk 1]
```diff
@@ -403,7 +408,8 @@ def notify_readers(self, edit, content_fields=[]):
         ## Notify action editors
         if is_public or self.get_action_editors_id(number=forum.number) in readers:
             message = f'''Hi {{{{fullname}}}},
-{before_invitation} {lower_formatted_invitation} {action} on a submission for which you are serving as Action Editor {forum.content['title']['value']}
+
+{before_invitation} {lower_formatted_invitation} on a submission you are an Action Editor for has been {action}
```
Comment (celestemartinez) [inline openreview/journal/journal.py:412]: (This sounds slight weird to me) Just a suggestion:

> {before_invitation} {lower_formatted_invitation} has been {action} on a submission for which you are an Action Editor

**Review by celestemartinez — APPROVED** [review]

File: openreview/journal/journal.py (hunk 1 above)
Comment (melisabok) [reply openreview/journal/journal.py:412]: The change is a suggestion from @zbialecki , I like your suggestion tough

Comment (melisabok) [reply openreview/journal/journal.py]: I changed the text with Celeste' suggestion, it is ready to merge now.

File: openreview/journal/journal.py [hunk 2]
```diff
@@ -394,7 +398,8 @@ def notify_readers(self, edit, content_fields=[]):
                     reviewer_recipients.append(reader)
         if reviewer_recipients:
             message = f'''Hi {{{{fullname}}}},
-{before_invitation} {lower_formatted_invitation} {action} on a submission for which you are serving as reviewer {forum.content['title']['value']}
+
+{before_invitation} {lower_formatted_invitation} has been {action} on a submission you are a reviewer.
```
Comment (celestemartinez) [inline openreview/journal/journal.py:402]:
```suggestion
{before_invitation} {lower_formatted_invitation} has been {action} on a submission for which you are a reviewer.
```

**Pull request merged**